spec/eyaml/util_spec.rb

```diff
@@ -4,7 +4,21 @@
   describe ".pretty_yaml" do
     it "will return a hash as YAML without the three dash prefix" do
       yaml_without_prefix = File.read(fixtures_root.join("pretty.yml"))
-      expect(EYAML::Util.pretty_yaml({"a" => "1", "b" => "2"})).to eq(yaml_without_prefix)
+      expect(EYAML::Util.pretty_yaml({"a"=>"1", "b"=>"2", "_c"=>{"_d"=>"3"}})).to eq(yaml_without_prefix)
+    end
+  end
+
+  describe ".with_deep_deundescored_keys" do
```

Comment:
**thought:** I'm wondering what would happen if we define `_underscored_secret` and `underscored_secret` in YAML?

Reply:
Previously it would raise

Reply:
It seems reasonable to raise? Why are we changing it? We are likely to replace one value with another - and we forgot to remove encrypted on non-encrypted value - would be great to know about it.